### PR TITLE
feature: Add docker-compose to run the module

### PR DESCRIPTION
**Pull request by leonampd**
### Descrição

There's no easier way to run our module. So, this includes
docker-compose file and a .env file to allow developers to run the
module in development environments with docker containers. This
containers are based on our [magento
image](https://github.com/pagarme/docker-library/tree/master/magento)

### Número da Issue

Não há issues relacionadas

### Testes Realizados

Realizado teste manual para validar a instalação do módulo na loja.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+version: '2'
+
+services:
+    selenium:
```
Comment (henriquekano) [inline docker-compose.yml]: The `selenium` and `firefox` services are not used.

**Comment by lflimeira** [comment]
@leonampd can you fix the changes that @henriquekano  proposed?

**Comment by leonampd** [comment]
@henriquekano thanks for the comment. I removed the unnecessary code.

File: README.md [hunk 2]
```diff
@@ -34,8 +34,25 @@
...
+### Instalando o Magento Community 1.x
+
+1. Execute o comando `docker-compose up -d` para iniciar os containers e a instalação do Magento
+2. Execute o comando `docker-compose logs -f magento` para acompanhar o processo de instalação.
+
+### Acessando a loja virtual através do navegador
+
+1. Altere seu arquivo `/etc/hosts` adicionando a entrada `127.0.0.1 magento`
+2. Acesse a loja no navegador utilizando o endereço `http://magento`
+
```
Comment (henriquekano) [inline README.md]: Add the default admin login and password

**Pull request merged**